c2corg_ui/static/js/documentediting.js

```diff
@@ -54,7 +55,14 @@ app.module.directive('appDocumentEditing', app.documentEditingDirective);
  * @export
  */
 app.DocumentEditingController = function($scope, $element, $attrs,
-    appAuthentication, appAlerts, appApi, authUrl) {
+    appAuthentication, appAlerts, appApi, authUrl, apiUrl, ngeoLocation) {
```

Comment:
I've never really taken care of the order of parameters nor it's alphabetical declaration in main.js and I never got an error, hopefully. Don't change that! :dancer: